src/Listeners/SaveVotesToDatabase.php

```diff
@@ -214,7 +213,8 @@ public function pushNewVote(Vote $vote)
     public function assertNotFlooding($actor)
     {
         if ($actor->last_vote_time !== null && Carbon::parse($actor->last_vote_time)->greaterThanOrEqualTo(Carbon::now()->subSeconds(10))) {
-            throw new FloodingException();
+            return False;
```

Comment:
Why are we no longer throwing a flooding exception? Won't returning true/false just make it seem like the request was successful when it did nothing?

Also - use `true` & `false` instead of `True` and `False`.

I've not tested this locally, was just taking a glance at the code to have a good idea once I review this.

Reply:
oops! my bad. This is what happen when you work with Python everyday.
I'll fix it.

Reply:
when i test it with the rate limit turn on, the exception will just throw too frequently, the log file got filled, i don't think its necessary.

Reply:
We'd definitely want to make the ratelimit interval able to be modified, but we don't want to get rid of it.